### PR TITLE
fixes-#6198-vsphere-provider-improve-documentation

### PR DESCRIPTION
**Pull request by tommymalmqvist**
There is some confusion about using the resource_pool arument to the resource definition. This will add a more complete example.

issue 6198

https://github.com/hashicorp/terraform/issues/6198

### CODE REVIEW

File: website/source/docs/providers/vsphere/index.html.markdown [hunk 1]
```diff
@@ -41,6 +41,10 @@ resource "vsphere_virtual_machine" "web" {
   folder = "${vsphere_folder.frontend.path}"
   vcpu   = 2
   memory = 4096
+  domain = "${var.vsphere_domain}"
+  datacenter = "${var.vsphere_datacenter}"
+  cluster = "${var.vsphere_cluster}"
+  resource_pool = "${var.vsphere_cluster}/Resources/${var.vsphere_resource_pool}"
```
Comment (chrislovecnm) [inline website/source/docs/providers/vsphere/index.html.markdown]: AWESOME!! How about some english describing how this works?? How about when there is not a cluster?  Any other use cases?

**Comment by chrislovecnm** [comment]
@tommymalmqvist any comments on my nit picks?

**Comment by tommymalmqvist** [comment]
hey @chrislovecnm, I'll add some more text tonight.

**Comment by chrislovecnm** [comment]
Thanks @tommymalmqvist!

@slack72 @jen20 LGTM

**Comment by chrislovecnm** [comment]
@stack72 thanks - but what u doin working on a Sunday :P Animal!!

**Pull request merged**